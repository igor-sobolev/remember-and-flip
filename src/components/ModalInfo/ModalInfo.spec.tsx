import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { ModalInfo } from '.';

describe('given ModalInfo component', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<ModalInfo>Example notification</ModalInfo>);
  });

  it('should match the snapshot', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
