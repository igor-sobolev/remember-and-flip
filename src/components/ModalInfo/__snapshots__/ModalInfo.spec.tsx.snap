// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`given ModalInfo component should match the snapshot 1`] = `
<div
  className="modal-info"
>
  <h3
    className="modal-info__message"
  >
    Example notification
  </h3>
</div>
`;
